modules/gapi/src/executor/gexecutor.cpp

```diff
@@ -155,15 +155,18 @@ void cv::gimpl::GExecutor::run(cv::gimpl::GRuntimeArgs &&args)
 #if !defined(GAPI_STANDALONE)
             // Building as part of OpenCV - follow OpenCV behavior
             // if output buffer is not enough to hold the result, reallocate it
-            auto& out_mat   = *get<cv::Mat*>(args.outObjs.at(index));
-            createMat(desc, out_mat);
-#else
+            if (cv::util::holds_alternative<cv::Mat*>(args.outObjs.at(index)))
+            {
+                auto& out_mat   = *get<cv::Mat*>(args.outObjs.at(index));
+                createMat(desc, out_mat);
+                continue;
+            }
+#endif // !defined(GAPI_STANDALONE)
             // Building standalone - output buffer should always exist,
             // and _exact_ match our inferred metadata
```

Comment:
So after #ifdef scope is changed, this comment is not correct anymore and meta is checked always.

Reply:
Fixed